source/server/config/http/dynamo.cc

```diff
@@ -8,13 +8,13 @@ namespace Envoy {
 namespace Server {
 namespace Configuration {
 
-HttpFilterFactoryCb DynamoFilterConfig::tryCreateFilterFactory(HttpFilterType type,
-                                                               const std::string& name,
-                                                               const Json::Object&,
-                                                               const std::string& stat_prefix,
-                                                               Server::Instance& server) {
-  if (type != HttpFilterType::Both || name != "http_dynamo_filter") {
-    return nullptr;
+HttpFilterFactoryCb DynamoFilterConfig::createFilterFactory(HttpFilterType type,
+                                                            const Json::Object&,
+                                                            const std::string& stat_prefix,
+                                                            Server::Instance& server) {
+  if (type != HttpFilterType::Both) {
```

Comment:
@htuch cannot think about anything else, but fwiw, currently, no HTTP filters support this behavior.